srcpkgs/dotool/INSTALL.msg

```diff
@@ -0,0 +1,6 @@
+A udev rule has been added to allow users in group input to use dotool
+without root permissions.  You can make it effective without rebooting
+by running:
+
+	$ sudo udevadm trigger
+
```

Comment:
not necessary, this isn't specific to void. maybe you'd want to add this to your documentation upstream.